Backend/lib/beckn-spec/src/Beckn/Types/Core/Taxi/Rating.hs

```diff
@@ -25,6 +25,6 @@ import EulerHS.Prelude hiding (id)
 data RatingMessage = RatingMessage
   { id :: Text,
     value :: Int,
-    feedback_form :: FeedbackForm
+    feedback_form :: [FeedbackForm]
```

Comment:
@Shikhar-24 Did you test by hitting BPP directly and sending `feedback_form` as an Object without Array ?